get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc

```diff
@@ -0,0 +1,147 @@
+= Neo4j Desktop User Interface Guide
+:slug: guide-neo4j-desktop
+:level: Beginner
+:toc:
+:toc-placement!:
+:toc-title: Overview
+:toclevels: 1
+:section: Get Started
+:section-link: get-started
+:experimental:
+:neo4j-version: 3.3.4
+
+.Goals
+[abstract]
+This article demonstrates how to use Neo4j for desktop for managing instances of Neo4j locally for development.
+
+.Prerequisites
+[abstract]
+Please have link:/download[Neo4j^] downloaded and installed.
+It also helps if you have read the section on link:/developer/get-started/graph-database[graph databases^].
+
+[role=expertise]
+{level}
+
+toc::[]
+
+//:img: .
+
+== Installing and Starting Neo4j Desktop
+
+If you haven't already, http://neo4j.org/download[download^] Neo4j. 
+Using the provided instructions (shown when downloading), follow Step 1 to install and start the Neo4j Desktop application.
+Instructions can also be found in the readme (`bin/neo4j start`) after unzipping the downloaded file.
```

Comment:
remove this line

Reply:
The line on "Instructions can also be found..."?